core/templates/domain/learner_dashboard/learner-playlist-modal.component.spec.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for for learnerPlaylistModal.
+ */
+
+import { async, ComponentFixture, fakeAsync, flushMicrotasks, TestBed } from
+  '@angular/core/testing';
```

Comment:
You can put this on the above line itself.

Reply:
done!